src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/MissingJavadocTypeCheck.java

```diff
@@ -127,21 +128,25 @@
  * </pre>
  * <p>
  * Example that allows missing comments for classes annotated with {@code @SpringBootApplication}
- * and {@code @Configuration}:
+ * , {@code @Configuration} and {@code @MyClass.Annotation}:
  * </p>
  * <pre>
  * &#64;SpringBootApplication // no violations about missing comment on class
  * public class Application {}
  *
  * &#64;Configuration // no violations about missing comment on class
  * class DatabaseConfiguration {}
+ *
+ * &#64;MyClass.Annotation // no violations about missing comment on class
+ * class SomeClass {}
```

Comment:
this example intention is to be real life example.

Lets move your example to separate cong+code block to show nuance of match.
```
@Annotation // ok
@MyClass.Annotation // ok
com.mycompany.MyClass.Annotation // violation, as this form is missed in config
```

Reply:
I need some help. How do I get rid of this CI failure? I understand it is complaining that I pushed a commit with changes not staged for commit. I tried `git commit --amend --allow-empty` but no success:
```
Please clean up or update .gitattributes file.
Git status output:
```

Reply:
I am not sure how you came to this.
I recommend to create new branch, reply patch or cherry pick or ..., remove on your local old branch, rename new branch to old branch, push with force to GitHub.

Reply:
It seems that running `mvn clean verify` modifies the file:
![Screenshot from 2022-07-13 19-24-22](https://user-images.githubusercontent.com/23459549/178786018-9abbcb90-0b9e-48ed-87e2-10d1371c437c.png)
![Screenshot from 2022-07-13 19-36-36](https://user-images.githubusercontent.com/23459549/178786028-f54aa978-0778-4a33-a816-b13967901704.png)

Is this a bug or a feature where `-`(dash) is not allowed in `.xml` files? I see other `.xml` containing such.

Reply:
I will try to escape it with `&#8211;` and see what happens.

Reply:
Same happens with `&#8211;`. `mvn clean verify` deletes it. I will just remove it.

Reply:
Done!